lib/utils/user_colors.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+
+const green = Colors.green;
+const lightGreen = Colors.lightGreen;
+const lime = Colors.lime;
+const yellow = Colors.yellow;
+const amber = Colors.amber;
+const orange = Colors.orange;
+const deepOrange = Colors.deepOrange;
+const red = Colors.red;
+const pink = Colors.pink;
+const purple = Colors.purple;
+const deepPurple = Colors.deepPurple;
+const cyan = Colors.cyan;
+const blue = Colors.blue;
+const lightBlue = Colors.lightBlue;
+const grey = Colors.grey;
+const brown = Colors.brown;
+const white = Colors.white;
```

Comment:
Dodaj plik listę zawierającą wszystkie powyższe kolory 😀

Reply:
Zależało mi żeby dane kolory miały swoje nazwy, ale rozumiem, że wolisz tak jak poniżej, wtedy żeby przywołać dany kolor z listy będziemy podawać jego pozycje, czyli jeśli chcemy 'Colors.green' to będzie to [0]:

List userColors = const [
Colors.green, 
Colors.lightGreen, 
Colors.lime, 
Colors.yellow, 
... ];

Reply:
czyli rozumiem, że mam zmienić na List userColors... jak wyżej? :)

Reply:
Tak